integration/control_plane.go

```diff
@@ -57,3 +60,17 @@ func (f *ControlPlane) KubeCtl() *KubeCtl {
 	k.Opts = append(k.Opts, fmt.Sprintf("--server=%s", f.APIURL()))
 	return k
 }
+
+func (f *ControlPlane) RESTClientConfig() *rest.Config {
+	c := &rest.Config{
+		Host: f.APIURL().String(),
+		ContentConfig: rest.ContentConfig{
+			NegotiatedSerializer: scheme.Codecs.WithoutConversion(),
```

Comment:
do we actually need this?

Reply:
It is required.
See: https://github.com/kubernetes/client-go/blob/ecd4f6fb8c21f85d3a0c0143fcbf83c662b6d106/rest/config.go#L284